### PR TITLE
Add multiple origin CORS example code

### PR DESCRIPTION
**Pull request by joech4n**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -801,6 +801,29 @@ There's a couple of things to keep in mind when enabling cors for a view:
...
+  Example:
+  
+.. code-block:: python
+
+    from chalice import Chalice, Response
+    app = Chalice(app_name='multipleorigincors')
+ 
+    @app.route('/cors_multiple_origins', methods=['GET'])
+    def supports_cors_multiple_origins():
+        origin = app.current_request.to_dict()['headers']['origin']
```
Comment (stealthycoin) [inline README.rst]: If no origin is supplied this will 500.

File: README.rst [hunk 2]
```diff
@@ -801,6 +801,29 @@ There's a couple of things to keep in mind when enabling cors for a view:
...
+        if origin == 'http://allowed1.example.com':
+            return Response(body='You sent a whitelisted origin!',
+                status_code=200,
```
Comment (stealthycoin) [inline README.rst]: Lots of PEP8 violations here.

File: README.rst [hunk 3]
```diff
@@ -801,6 +801,29 @@ There's a couple of things to keep in mind when enabling cors for a view:
...
+                headers={'Access-Control-Allow-Origin': 'http://allowed1.example.com'})
+        elif origin == 'http://allowed2.example.com':
```
Comment (stealthycoin) [inline README.rst]: This is mostly a repeat of the above if, this can be factored out to be a lot clearer.

File: README.rst [hunk 4]
```diff
@@ -801,6 +801,29 @@ There's a couple of things to keep in mind when enabling cors for a view:
...
+            return Response(body='You sent a whitelisted origin!',
+                status_code=200,
```
Comment (stealthycoin) [inline README.rst]: This is the default status code so there is no need to pass this in, though it is nice to be explicit.

**Comment by joech4n** [comment]
@stealthycoin Thanks for the tips! I've gone ahead and updated the pull request with your suggestions.

**Pull request merged**